### PR TITLE
setup

### PR DESCRIPTION
**Pull request by Matthensley777**

### CODE REVIEW

**Comment by Matthensley777** [comment]
@morecallan part3

**Comment by Matthensley777** [comment]
@morecallan validate is working

File: partials/contacts/new.html [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
+      <div class="form-group">
+        <label>
+          Country:
+          <input type="text" name="Country" class="form-control" ng-model="newContact.Country" required>
+        </label>
+      </div>
+        <div class="form-group">
+        <label>
+          Phone Number:
+          <input type="text" name="PhoneNumber" class="form-control" ng-model="newContact.PhoneNumber" required>
```
Comment (morecallan) [inline partials/contacts/new.html]: Careful - it looks like this isn't it PhoneNumber consistently throughout your application (your view has contact.phone). Search through your application so you can make sure that the phone number saves to firebase the same way and the data is displayed the same way.

**Review by morecallan — COMMENTED** [review]
@Matthensley777 - great job with the data validation. The one thing I'm seeing is related to the phone number. Come see me for clarification :)

File: partials/contacts/new.html (hunk 1 above)
Comment (Matthensley777) [reply partials/contacts/new.html]: @morecallan fixed

**Comment by morecallan** [comment]
👍

**Pull request merged**